apps-contrib/resources/kafka-console-consumer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mainClass": "kafka.tools.ConsoleConsumer",
+  "repositories": [
+    "central"
+  ],
+  "dependencies": [
+    "org.apache.kafka::kafka:latest.stable"
```

Comment:
What about adding `org.slf4j:slf4j-nop:1.7.30` as a dependency too, to avoid the slf4j warning?

This gives
```bash
$ cs launch org.apache.kafka::kafka:latest.stable org.slf4j:slf4j-nop:1.7.30 -M kafka.tools.ConsoleConsumer -- --help
This tool helps to read data from Kafka topics and outputs it to standard output.
…
```
instead of
```bash
$ cs launch org.apache.kafka::kafka:latest.stable -M kafka.tools.ConsoleConsumer -- --help
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
This tool helps to read data from Kafka topics and outputs it to standard output.
…
```

Reply:
Added `org.slf4j:slf4j-nop:1.7.30`. I wonder if it is possible to use something like `latest.stable` for this part too...

Reply:
That could work, although there might be issues when running the application with `cs launch --contrib kafka-console-consumer:custom-version` IIRC. Let's keep `1.7.30` for now…